st2client/tests/base.py

```diff
@@ -86,6 +86,8 @@ def setUp(self):
             if var in os.environ:
                 del os.environ[var]
 
+        os.environ['ST2_SKIP_CLI_CONFIG'] = '1'
```

Comment:
Environment variables can also be specified on per command invocation basis. In fact, that's how I envisioned people would use it.

An alternative would be adding ``--skip-config`` flag. I'm fine with that.